deepdow/utils.py

```diff
@@ -122,6 +122,10 @@ def check_indices_agree(*frames):
 
 
 def prices_to_returns(prices, use_log=True):
+    # ? Maybe use a more meaningful name since
+    # ? not all variables to be transformed are prices
+
+    # TODO: Add other methods like pct_change
```

Comment:
You are totally right, the name is misleading.

Reply:
What the function is really doing is computing the difference, converting the time-series to stationary. [Here](https://machinelearningmastery.com/remove-trends-seasonality-difference-transform-python/#:~:text=Stationary%20Time%20Series,-The%20observations%20in&text=Time%20series%20are%20stationary%20if,can%20be%20easier%20to%20model.) are some other techniques and naming that could be used.

Reply:
The other comment (line 128) was a mistake, revisiting that.